config/routes.rb

```diff
@@ -1,4 +1,4 @@
-Rails.application.routes.draw do
+ Rails.application.routes.draw do
```

Comment:
I'm not sure why this file changed -- can we revert the changes? In general, any file that does not have functional changes relating to the feature branch should not be `git add`ed before committing. To roll this back to an earlier version, you'll have to do a `git checkout` on the file to bring the older version of the file back into your feature branch, then `git add` it and `git commit`:

See 'Checking Out a File' from https://www.atlassian.com/git/tutorials/undoing-changes/git-checkout.
See also http://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git.

You could do the same thing for `app/views/questions/index.html.erb` (mentioned above).